modules/signals/migrations/migration.json

```
@@ -1,4 +1,10 @@
 {
   "$schema": "../../../node_modules/@angular-devkit/schematics/collection-schema.json",
-  "schematics": {}
+  "schematics": {
+    "ngrx-signals-migration-18-beta-disable-protected-state": {
+      "description": "Make all existing Signal Stores unprotected",
+      "version": "18.0.0-beta",
+      "factory": "./18_0_0-beta-disable-protected-state/index"
+    }
```

Great work Rainer! 👏 

Let's just specify the exact next version and use the version as a prefix (https://github.com/ngrx/platform/pull/4439#discussion_r1685716553):

```suggestion
    "18_0_0-rc_3-protected-state": {
      "description": "Make the states of all SignalStores unprotected",
      "version": "18.0.0-rc.3",
      "factory": "./18_0_0-rc_3-protected-state/index"
    }
```